packages/amplication-server/src/core/build/dto/BuildContext.ts

```diff
@@ -0,0 +1,8 @@
+import { BuildContextData } from './BuildContextData';
+
+export interface BuildContext {
```

Comment:
We have 5 interfaces here that are 1:1 matching with the Step 2 service (`build-manager-service`). This means that any change we make on either side will cause a silent failure (we don't really test this...)

I'd like us to move this to a shared lib in our monorepo. This will help us prevent failures and allow us to share this code with a single source of truth. Let's utilize lerna ;)

Reply:
@mshidlov It's a project level decision that has to be taken.

Reply:
@EugeneTseitlin Currently, to not delay the delivery the commons will be duplicated, and we will back to it later. This is an important issue and cross-packages decision let's have a discussion and decide on a shared modals strategy.